src/models/room.ts

```diff
@@ -1968,11 +1968,11 @@ export class Room extends ReadReceipt<RoomEmittedEvents, RoomEventHandlerMap> {
         ));
     }
 
-    private updateThreadRootEvents = (thread: Thread, toStartOfTimeline: boolean): void => {
+    private updateThreadRootEvents = (thread: Thread, toStartOfTimeline: boolean, recreateEvent: boolean): void => {
```

Comment:
I can't help but wonder why some functions are defined with regular `private func(args) { .... }` method syntax, and others are defined as properties with `private func = (args) => {...}`. But I guess that's not new.

Reply:
[Arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) have the benefit of being strongly bound to the context of the class. So even if you pass the method as a callback/handler it'll have the correct `this` context, unlike `function` syntax (without `.bind`

Reply:
makes sense I guess, thanks.